cmd/analyze/main.go

```diff
@@ -206,3 +212,31 @@ func main() {
 		dynamicAnalysis(pkg)
 	}
 }
+
+func generateSSLCerts() error {
```

Comment:
Can this be moved into a small module under `internal`?

This would make it easier to reuse, test and refactor.

Reply:
Furthermore the cert generation needs to be added to `cmd/worker` as well.